docker/app/Dockerfile

```diff
@@ -62,8 +63,7 @@ RUN mv $PHP_INI_DIR/php.ini-development $PHP_INI_DIR/php.ini
 COPY --from=sillsdev/web-languageforge:wait-latest /wait /wait
 
 FROM ${ENVIRONMENT}-app AS languageforge-app
-ARG BUILD_VERSION=9.9.9
-ENV BUILD_VERSION=${BUILD_VERSION}
```

Comment:
Is the ENV BUILD_VERSION no longer needed?

Reply:
not as far as I could tell.  I'm not 100% sure it would've been there in the first place, maybe you have some history on it?

Reply:
ok thanks.